docs/installation/install.md

```diff
@@ -390,6 +390,13 @@ The installation of a development version of Liqo can be triggered specifying a
 In this case, *liqoctl* proceeds to **clone the repository** (either from the official repository, or from a fork configured through the `--repo-url` flag) at the given revision, and to leverage the Helm chart therein contained.
 Alternatively, the Helm chart can be retrieved from a **local path**, as configured through the `--local-chart-path` flag.
 
+Example:
+```bash
+liqoctl install <provider> --version <commit-sha> \
+    --repo-url <forked-repo-url> \
+    --local-chart-path <path-to-local-chart>
```

Comment:
The `--repo-url` and `--local-chart-path` are alternative. I would suggest to add two examples to show both possibilities (i.e., either automatically downloading the chart from a repository, or using an already existing local copy of the chart).